qdrant-landing/content/documentation/tutorials/chat-pdf-documents.md

```diff
@@ -0,0 +1,217 @@
+
+### Procedure
+
+Retrieval Augmented Generation (RAG) combines search with language generation. An external information retrieval system is used to identify documents likely to provide information relevant to the user's query. These documents, along with the user's request, are then passed on to a text-generating language model, producing a natural response.
+
+This method enables a language model to respond to questions and access information from a much larger set of documents than it could see otherwise. The language model only looks at a few relevant sections of the documents when generating responses, which also helps to reduce inexplicable errors.
+
+## Prerequisites
+First, install all dependencies:
+
+```python
+!pip install -U  \
+    llama-index  \
+    llama-parse \
+    python-dotenv \
+    llama-index-embeddings-jinaai  \
+    llama-index-llms-huggingface  \
+    llama-index-vector-stores-qdrant  \
+    "huggingface_hub[inference]"  \
+    datasets
+```
+
+Set up secret key values on `.env` file:
+
+```bash
+JINAAI_API_KEY
+HF_INFERENCE_API_KEY
+LLAMA_CLOUD_API_KEY
+QDRANT_HOST
+QDRANT_API_KEY
+```
+
+Load all environment variables:
+
+```python 
+import os
+from dotenv import load_dotenv
+load_dotenv('./.env')
+```
+## Implementation 
+
+### Connect Jina Embeddings and Mixtral LLM
+
+LlamaIndex provides built-in support for the [Jina Embeddings API](https://jina.ai/embeddings/#apiform). To use it, you need to initialize the `JinaEmbedding` object with your API Key and model name.
+
+For the LLM, you need wrap it in a subclass of `llama_index.llms.CustomLLM` to make it compatible with LlamaIndex.
+
+```python
+# connect embeddings
+from llama_index.embeddings.jinaai import JinaEmbedding
+
+jina_embedding_model = JinaEmbedding(
+    model="jina-embeddings-v2-base-en",
+    api_key=os.getenv("JINAAI_API_KEY"),
+)
+
+# connect LLM
+from llama_index.llms.huggingface import HuggingFaceInferenceAPI
+
+mixtral_llm = HuggingFaceInferenceAPI(
+    model_name = "mistralai/Mixtral-8x7B-Instruct-v0.1",
+    token=os.getenv("HF_INFERENCE_API_KEY"),
+)
+```
+
+### Prepare data for RAG
+
+This example will use household appliance manuals, which are generally available as PDF documents.
+
+In the `data` folder, we have three documents, and we will use LlamaParse to extract the textual content from the PDF and use it as a knowledge base in a simple RAG.
+
+The free LlamaIndex Cloud plan is sufficient for our example:
+
+```python
+import nest_asyncio
+nest_asyncio.apply()
+from llama_parse import LlamaParse
+
+llamaparse_api_key = os.getenv("LLAMA_CLOUD_API_KEY")
+
+llama_parse_documents = LlamaParse(api_key=llamaparse_api_key, result_type="markdown").load_data([
+    "data/DJ68-00682F_0.0.pdf", 
+    "data/F500E_WF80F5E_03445F_EN.pdf", 
+    "data/O_ME4000R_ME19R7041FS_AA_EN.pdf"
```

Comment:
Can we put some links to these files? Even a screenshot to show what are the files we work with.